erpnext/selling/doctype/sales_order_item/sales_order_item.py

```diff
@@ -12,4 +12,13 @@ def __setup__(self):
 		print_settings_for_item_table(self)
 
 def on_doctype_update():
-	frappe.db.add_index("Sales Order Item", ["item_code", "warehouse"])
\ No newline at end of file
+	frappe.db.add_index("Sales Order Item", ["item_code", "warehouse"])
+
+@frappe.whitelist()
+def get_customer_item_ref_code(item,customer_name):
```

Comment:
docstrings and comments

Reply:
@rvpasari Since its a small function, we can ignore docstrings.
If a function is difficult to understand or lengthy, we will need docstring

Reply:
Based on Sahil's recommendation, since it was an API endpoint I thought we were enforcing docstrings

Reply:
The function is just 5 lines and easy to understand, so we can do away with docstrings